### ISSUE TITLE
Who is better? Pokebuddy or pokefarmer? 

### ISSUE THREAD

**Issue by itzJOKINHA** [finndev/PokeBuddy#353]

**Comment by Deuxgod** [comment]
Are you kidding? Both are completely different despite having the same purpose. So use whatever you want and be thankfull that both are doing a great job for free.

**Comment by RobJansen** [comment]
I use pokebuddy but they are completely different

**Comment by itzJOKINHA** [comment]
Pokefarmer is not free, i bought it , i know they are differents but i want to know whp is better to use , who gives more exp etc

**Comment by IamDeric** [comment]
Pokebuddy for me, i bought pokefarmer last night also, and its keep stoping after 30mins but they have an updated the bot today also, once they upgrade the GUI features on pokebuddy ill stick on pokebuddy. I got more exp/hr in pokebuddy also.

**Comment by itzJOKINHA** [comment]
yeah me too, i bought pokefarmer yesterday and i prefer pokebuddy

**Comment by IamDeric** [comment]
If i just can refund the payment and just donate it to pokebuddy ill do it. 😂😂😂, but ill make the donation soon. For fixing this awsome bot.

**Comment by oOBlueArrowOo** [comment]
@itzJOKINHA u got ripped off xD

**Comment by Rab89** [comment]
the only thing i liked about pokefarmer was if you were softbanned it would stay spinning a pokestop until you got unbanned.
but hands down pokebuddy, far superior functioning speed and far superior UI
